gossip/src/main/java/io/zeebe/gossip/failuredetection/SyncController.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.zeebe.gossip.failuredetection;
+
+import io.zeebe.gossip.GossipConfiguration;
+import io.zeebe.gossip.GossipContext;
+import io.zeebe.gossip.Loggers;
+import io.zeebe.gossip.membership.Member;
+import io.zeebe.gossip.membership.MembershipList;
+import io.zeebe.gossip.membership.RoundRobinMemberIterator;
+import io.zeebe.gossip.protocol.GossipEvent;
+import io.zeebe.gossip.protocol.GossipEventSender;
+import io.zeebe.transport.ClientResponse;
+import io.zeebe.util.sched.ActorControl;
+import io.zeebe.util.sched.future.ActorFuture;
+import java.time.Duration;
+import org.agrona.DirectBuffer;
+import org.slf4j.Logger;
+
+public class SyncController {
+  private static final Logger LOG = Loggers.GOSSIP_LOGGER;
+  private static final Duration SYNC_INTERVAL = Duration.ofSeconds(15);
```

Comment:
remove this constant and use the configuration value instead

Reply:
yo sorry forgot this